lib/datasource/github-releases/index.ts

```diff
@@ -1,14 +1,21 @@
-import URL from 'url';
 import * as packageCache from '../../util/cache/package';
 import { GithubHttp } from '../../util/http/github';
+import { ensureTrailingSlash } from '../../util/url';
 import { GetReleasesConfig, ReleaseResult } from '../common';
 
 export const id = 'github-releases';
+export const defaultRegistryUrls = ['https://github.com'];
```

Comment:
Where is this used?

Reply:
on line L154 🙃 Is used if extracted urls is empty
https://github.com/renovatebot/renovate/blob/5896d9cb37447151f2e13d0b8570bc1618dfbca4/lib%2Fdatasource%2Findex.ts#L150-L163

Reply:
is hidden behind interface
https://github.com/renovatebot/renovate/blob/40ae43817a6713423a9e968296806614740bef01/lib%2Fdatasource%2Fcommon.ts#L73-L81